File: 0920-uncommon-words-from-two-sentences/0920-uncommon-words-from-two-sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        StringBuilder ss = new StringBuilder(s1 + " " + s2);
        HashMap<String, Integer> mp = new HashMap<>();
        for (String wrd: ss.toString().split(" ")) mp.put(wrd, mp.getOrDefault(wrd, 0) + 1);
        List<String> ans = new LinkedList<>();
        for (String wrd: mp.keySet()) 
            if (mp.get(wrd) == 1) ans.add(wrd);

        return ans.toArray(new String[ans.size()]);
    }
}

File: 0920-uncommon-words-from-two-sentences/0920-uncommon-words-from-two-sentences.kt
class Solution {
    fun uncommonFromSentences(s1: String, s2: String): Array<String> {
        val s_merge: Array<String> = (s1 + " " + s2).split(" ").toTypedArray();
        val mp: HashMap<String, Int> = hashMapOf();
        val ans: MutableList<String> = mutableListOf();
        for (wrd in s_merge) 
            mp[wrd] = mp[wrd]?.plus(1) ?: 1;
        for ((wrd, cnt) in mp) 
            if (cnt == 1) ans.add(wrd);
        return ans.toTypedArray();
    }
}